x-pack/legacy/plugins/actions/mappings.json

```diff
@@ -27,6 +27,9 @@
       },
       "apiKey": {
         "type": "binary"
+      },
+      "createdAt": {
```

Comment:
I haven't looked to see how this is used, beyond the PR comment that it's part of FT (to have tests identify their own garbage).  SO's already have an `updated_at`, I wonder if we could reuse that instead.  Perhaps the version in the SO is not searchable? Though, this isn't a bad field to have in the SO anyway, for diagnostic reasons.

Reply:
I did a check and it works! Since we're not updating the objects, there's no risk of interference. I went ahead and made the change to keep it clean https://github.com/elastic/kibana/pull/55227/commits/f0a03854817d88f45ccda7190ecb2376a0faac9c.